crates/types/src/v1alpha1/wasmcloud_host_config.rs

```diff
@@ -54,14 +54,30 @@ pub struct WasmCloudHostConfigSpec {
     /// The Jetstream domain to use for the NATS sidecar. Defaults to "default".
     #[serde(default = "default_jetstream_domain")]
     pub jetstream_domain: String,
+    /// Allow the host to deploy using the latest tag on OCI components or providers
+    #[serde(default)]
+    pub allow_latest: bool,
+    /// Allow the host to pull artifacts from OCI registries insecurely
+    #[serde(default)]
+    pub allowed_insecure: Option<Vec<String>>,
     /// The log level to use for the host. Defaults to "INFO".
     #[serde(default = "default_log_level")]
     pub log_level: String,
+    pub policy_service: Option<PolicyService>,
     /// Kubernetes scheduling options for the wasmCloud host.
     pub scheduling_options: Option<KubernetesSchedulingOptions>,
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct PolicyService {
+    pub subject: Option<String>,
```

Comment:
We call it `topic` in the wasmCloud host, should we match that language here?

Reply:
Probably, but nats doesn't actually have topics since they're called subjects. I can rename to match if we want.

Reply:
I think consistency with the rest of the tooling is probably the right thing to favor here